godot-core/src/registry/plugin.rs

```diff
@@ -104,10 +106,14 @@ pub enum PluginItem {
         ///
         /// Always present since that's the entire point of this `impl` block.
         register_methods_constants_fn: ErasedRegisterFn,
+        /// Method, Signal, Constant documentation.
+        docs: [&'static str; 3],
     },
 
     /// Collected from `#[godot_api] impl I... for MyClass`.
     ITraitImpl {
+        /// Virtual method documentation.
+        docs: &'static str,
```

Comment:
since this is a single string, you can give it a better name than just `docs`. maybe `method_docs`?